### ISSUE TITLE
Request: suggested action color

### ISSUE THREAD

**Issue by Feichtmeier** [nana-4/material-ubuntu-theme#1]
How do you think about chosing the suggested action color that the ambiance theme uses? Just an idea :)

![bildschirmfoto-20171113113905-606x442](https://user-images.githubusercontent.com/15329494/32721626-5ee38452-c867-11e7-901e-6e151a04b558.png)
![bildschirmfoto-20171113113945-606x447](https://user-images.githubusercontent.com/15329494/32721627-5efa7c02-c867-11e7-9b88-25980d0c887d.png)

**Comment by nana-4** [comment]
Hi, thanks for your idea. :)

Just my idea; but I personally thought the green color of Ambiance is quite odd and irregular, so I changed it to Ubuntu Orange.

However, if many people think that green is better, I'm willing to change it.

So, I'd like to hold this thread and listen to people's opinions.

**Comment by Feichtmeier** [comment]
Yeah, it can look out of place. But it makes sense somehow and they had it in unity8 too.
This materia version is awesome by the way :) Especially that the title bars adapt to their focus (like they do in adwaita and ambiance). Most themes are bad at this.

**Comment by ghost** [comment]
I agree with OP, green for this button is what we are used to see in Ubuntu since several years.

Excellent theme btw, didn't see this one before even if I often watch your work @nana-4.

I don't use Ubuntu anymore since 2009 but I always liked Ambiance. Installing right now...
